sdk/apimanagement/api-management-custom-widgets-tools/src/node/readdir.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import fs from "fs";
+import path from "path";
+
+function readdir(dir: string, root: string): string[] {
+  const results: string[] = [];
+
+  fs.readdirSync(root + dir).forEach((file) => {
```

Comment:
I'd recommend making `readdir` async and consume "fs/promises"

Reply:
I don't see any benefits in doing so as it's used exactly on just one place and it would look like
`const files = await readdir("", rootLocal);`